src/test/components/Root-history.test.js

```diff
@@ -103,7 +103,7 @@ describe('Root with history', function() {
 
     expect(window.history.length).toBe(1);
 
-    expect(getByText('Downloading and processing the profile...')).toBeTruthy();
+    expect(getByText('Downloading and processing the profile…')).toBeTruthy();
```

Comment:
@julienw according to flod's suggestion, I replaced `...` with this single unicode character `…` to match the snapshots.